apps/mark/frontend/src/apimachine_config.test.tsx

```diff
@@ -26,7 +26,7 @@ test('machineConfig is fetched from api client by default', async () => {
   apiMock.expectGetElectionState({
     precinctSelection: ALL_PRECINCTS_SELECTION,
   });
-  render(<App reload={jest.fn()} apiClient={apiMock.mockApiClient} />);
+  render(<App reload={vi.fn()} apiClient={apiMock.mockApiClient} />);
   apiMock.setAuthStatusPollWorkerLoggedIn(electionDefinition);
-  await screen.findByText('mock-code-version');
+  await vi.waitFor(() => screen.getByText('mock-code-version'));
```

Comment:
I think I ran into something similar and a workaround I found was to use `vi.useFakeTimers({ shouldAdvanceTime: true })`. I think it's worth trying that out since `findBy*` is a very useful and readable pattern.

Alternatively, do we need fake timers everywhere?